lib/screens/translation_screen.dart

```diff
@@ -0,0 +1,360 @@
+import 'dart:io' show File, Directory;
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:async/async.dart';
+import 'package:intl/intl.dart' show DateFormat;
+import 'package:flutter_sound/flutter_sound.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:flushbar/flushbar.dart';
+
+import '../models/sentence_model.dart';
+import '../models/translation_model.dart';
+import '../l10n/strings.dart';
+import '../config.dart';
+import '../services/translation_service.dart';
+import '../services/sentence_service.dart';
+import 'error_screen.dart';
+
+class TranslationScreen extends StatefulWidget {
+  @override
+  State createState() {
+    return TranslationScreenState();
+  }
+}
+
+class TranslationScreenState extends State<TranslationScreen> {
+  List<Sentence> _sentences;
+
+  String _recorderTimer;
+  bool _isRecording;
+  double _recorderIconSize = 60.0;
+  FlutterSound _flutterSound;
+  StreamSubscription _recorderSubscription;
+  String _author = dummyUserName;
+  String _targetLang;
+  String _sourceLanguage;
+
+  Sentence _currentSentence;
+  int _progression = 0;
+  PageController _pageController;
+
+  final Map<String, Color> _isRecordingColor = {
+    "accent": Colors.red[600],
+    "primaryLight": Colors.red[100],
+    "primaryDark": Colors.red[800]
+  };
```

Comment:
Good job creating a map to hold this. Instead of a map from string to Color, I would prefer a Map from an enum to Color. _isRecordingColor[ColorTheme.primaryLight] is better than _isRecordingCorler["primaryLight"]